UltiSnips/robot.snippets

```diff
@@ -0,0 +1,354 @@
+snippet psct "Page Should Contain Textfield"
+Page Should Contain Textfield    ${1:${locator}}    ${2:${message=}}    ${3:loglevel=INFO}
+endsnippet
+
+snippet psnc "Page Should Not Contain"
+Page Should Not Contain    ${1:${text}}    ${2:loglevel=INFO}
+endsnippet
+
+snippet psncb "Page Should Not Contain Button"
+Page Should Not Contain Button    ${1:${locator}}    ${2:${message=}}    ${3:loglevel=INFO}
+endsnippet
+
+snippet psncc "Page Should Not Contain Checkbox"
+Page Should Not Contain Checkbox    ${1:${locator}}    ${2:${message=}}    ${3:loglevel=INFO}
+endsnippet
+
+snippet psnce "Page Should Not Contain Element"
+Page Should Not Contain Element    ${1:${locator}}    ${2:${message=}}    ${3:loglevel=INFO}
+endsnippet
+
+snippet psnci "Page Should Not Contain Image"
+Page Should Not Contain Image    ${1:${locator}}    ${2:${message=}}    ${3:loglevel=INFO}
+endsnippet
+
+snippet psncl "Page Should Not Contain Link"
+Page Should Not Contain Link    ${1:${locator}}    ${2:${message=}}    ${3:loglevel=INFO}
+endsnippet
+
+snippet psncrb "Page Should Not Contain Radio Button"
+Page Should Not Contain Radio Button    ${1:${locator}}    ${2:${message=}}    ${3:loglevel=INFO}
+endsnippet
+
+snippet psnct "Page Should Not Contain Textfield"
+Page Should Not Contain Textfield    ${1:${locator}}    ${2:${message=}}    ${3:loglevel=INFO}
+endsnippet
+
+snippet rktrof "Register Keyword To Run On Failure"
+Register Keyword To Run On Failure    ${1:${kw}}
+endsnippet
+
+snippet wuec "Wait Until Element Contains"
+Wait Until Element Contains    ${1:${locator}}    ${2:${text}}    ${3:${timeout=None}}    ${4:${error=None}}
+endsnippet
+
+snippet wuednc "Wait Until Element Does Not Contain"
+Wait Until Element Does Not Contain    ${1:${locator}}    ${2:${text}}    ${3:${timeout=None}}    ${4:${error=None}}
+endsnippet
+
+snippet wueie "Wait Until Element Is Enabled"
+Wait Until Element Is Enabled    ${1:${locator}}    ${2:${timeout=None}}    ${3:${error=None}}
+endsnippet
+
+snippet wueinv "Wait Until Element Is Not Visible"
+Wait Until Element Is Not Visible    ${1:${locator}}    ${2:${timeout=None}}    ${3:${error=None}}
+endsnippet
+
+snippet wueiv "Wait Until Element Is Visible"
+Wait Until Element Is Visible    ${1:${locator}}    ${2:${timeout=None}}    ${3:${error=None}}
+endsnippet
+
+snippet wupc "Wait Until Page Contains"
+Wait Until Page Contains    ${1:${text}}    ${2:${timeout=None}}    ${3:${error=None}}
+endsnippet
+
+snippet wupce "Wait Until Page Contains Element"
+Wait Until Page Contains Element    ${1:${locator}}    ${2:${timeout=None}}    ${3:${error=None}}
+endsnippet
+
+snippet wupdnc "Wait Until Page Does Not Contain"
+Wait Until Page Does Not Contain    ${1:${text}}    ${2:${timeout=None}}    ${3:${error=None}}
+endsnippet
+
+snippet wupdnce "Wait Until Page Does Not Contain Element"
+Wait Until Page Does Not Contain Element    ${1:${locator}}    ${2:${timeout=None}}    ${3:${error=None}}
+endsnippet
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Could you remove these trailing newlines please? :)

Reply:
Sure, I didn't notice them :)
done